prober/grpc_health_v1.go

```diff
@@ -0,0 +1,50 @@
+package prober
+
+import (
+	"context"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/prometheus/blackbox_exporter/config"
+
+	"google.golang.org/grpc"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+func ProbeGRPCHEALTHV1(ctx context.Context, target string, module config.Module, registry *prometheus.Registry, logger log.Logger) bool {
+	var (
```

Comment:
typically in Go, you do not define your variables at the top of the function, but where you first use them. And you often use `:=` so you don't need to specify the types:

```
parts := strings.Split(target, "/")
host, service := parts[0], parts[1]

conn, err := grpc.Dial(host, grpc.WithInsecure())
if err != nil {
...
}
...
```